### PR TITLE
Show only operator at release

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:

/kind bug
/kind infrastructure

**What this PR does / why we need it**:
Right now we show the handler and also we don't include the CRDs, the
release has to show just the operator manifest and image since handler
is deployed by it.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,13 +167,16 @@ prepare-major: $(RELEASE_NOTES)
...
+release: OPERATOR_IMAGE_SUFFIX =: $(shell hack/version.sh)
+release: HANDLER_IMAGE_SUFFIX =: $(shell hack/version.sh)
+release: manifests push-handler push-operator $(description) $(GITHUB_RELEASE) version/version.go
 	DESCRIPTION=$(description) \
 	GITHUB_RELEASE=$(GITHUB_RELEASE) \
 	TAG=$(shell hack/version.sh) \
 				   hack/release.sh \
-						$(shell find $(MANIFESTS_DIR) -type f)
+						$(shell find $(MANIFESTS_DIR) -type f) \
+						deploy/crds/nmstate.io_nmstates_crd.yaml \
```
Comment (phoracek) [inline Makefile:178]: Why do we need these two CRDs? Aren't they in the MANIFESTS_DIR already?

Comment (qinqon) [reply Makefile:178]: Nope, at manifests we have only the rendered stuff, CRDs are not passed to hack/render-manifests.go

Comment (phoracek) [reply Makefile:178]: Gotcha, thanks

**Pull request merged**